src/application/services/useNote.ts

```diff
@@ -190,7 +187,8 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
     }
 
     await noteService.updateNoteContent(currentId.value, content);
-    note.value = { ...note.value, content };
+    note.value = { ...note.value,
+      content };
```

Comment:
bad formatting. It should format such cases as 

```suggestion
    note.value = { 
      ...note.value,
      content 
    };
```

Reply:
we can create an issue in the Config for that

Reply:
done